.github/workflows/publish-dist.yml

```diff
@@ -8,57 +8,34 @@ jobs:
   run:
     if: github.event.workflow_run.conclusion == 'success'
     runs-on: ubuntu-latest
+    timeout-minutes: 10
     steps:
       - name: Download build-dist artifacts
         uses: dawidd6/action-download-artifact@v2
```

Comment:
I'm really going to insist on this point: please pin this down to a specific sha, at the very least.

Reply:
Done in a separate commit, like in cockpit-machines in https://github.com/cockpit-project/cockpit-machines/pull/143